php/content.md

```diff
@@ -68,6 +68,15 @@ Where `src/` is the directory containing all your php code and `config/` contain
 
 We provide the helper scripts `docker-php-ext-configure`, `docker-php-ext-install`, and `docker-php-ext-enable` to more easily install PHP extensions.
 
+In order to keep the images smaller, the php source is kept in a compressed tar file. To facilitate linking of the php source with any extension, we also provide the helper script `docker-php-source` to easily extract the tar or delete the extracted source. Note: if you do use `docker-php-source` to extract the source, be sure to delete it in the same layer of the docker image.
+
+```Dockerfile
+FROM php:5-apache
```

Comment:
This should be `5.6` like all the rest so that we encourage users to use `X.Y`. :+1: